pages/installation/os/centos.rst

```diff
@@ -92,6 +92,7 @@ The last step is to enable Graylog during the operating system's startup::
   $ sudo systemctl enable graylog-server.service
   $ sudo systemctl start graylog-server.service
 
+The next step is it :ref:`ingest messages <ingest_data>` into your Graylog and extract the messages with :ref:`extractors <extractors>` or use :ref:`the Pipelines <pipelinestoc>` to work with the messages.
```

Comment:
I think this should be "The next step is to ingest messages into Graylog and ...".

Same in the other files.